plugins/modules/cloud/smartos/vmadm.py

```diff
@@ -78,6 +78,7 @@
     description:
       - A list of disks to add, valid properties are documented in vmadm(1M).
```

Comment:
I'd really wish the documentation here would be slightly more helpful.

Reply:
Couldn't agree more. BOTMETA says this module is owned by $team_solaris. Maybe they can put a more thorough documentation in here?

I found the information to be in:
https://smartos.org/man/1m/vmadm

But I don't think this should be the PR for that.